### ISSUE TITLE
Add an option to shush the AGP compatibility warning

### ISSUE THREAD

**Issue by liutikas** [autonomousapps/dependency-analysis-gradle-plugin#1194]
**Is your feature request related to a problem? Please describe.**

Today, if you sue dependency analysis gradle plugin with the latest version of AGP you will always get
```
The Dependency Analysis plugin is only known to work with versions of AGP between 8.0.0 and 8.4.0-alpha01. You are using 8.6.0-alpha03. Proceed at your own risk.
```

**Describe the solution you'd like**

I'd like to have an option of disabling this warning. Gradle property or DSL would work great.

**Describe alternatives you've considered**

We currently added this warning to our warning allowlist system, sadly it print out every time configuration is run even if an unrelated task is run

**Additional context**
AndroidX tends to use the latest version of AGP for various integration reasons, so we cannot hold back until dependency-analysis-gradle-plugin is updated. We'd like to proceed at our own risk and have no warnings.

**Comment by eygraber** [comment]
I think you can add `dependency.analysis.compatibility=NONE` to `gradle.properties`

**Comment by pvdmde** [comment]
Please just update the compatibility matrix for the .1 patch

**Comment by MGaetan89** [comment]
> Please just update the compatibility matrix for the .1 patch

It was already done in https://github.com/autonomousapps/dependency-analysis-gradle-plugin/pull/1195, but not yet released.

**Comment by autonomousapps** [comment]
Resolved already, just use https://github.com/autonomousapps/dependency-analysis-gradle-plugin/issues/1194#issuecomment-2142970441
